### DISCUSSION TITLE
Easy development of modules/interfaces/....

### DISCUSSION THREAD

**Discussion by maartenvn** [directus/directus#3044] [Feature Requests]
Add a command to the CLI that would spin up a temporary Directus instance (with SQLite as database) with support for hot-reloading to develop/test modules/interfaces/....

**Comment by rijkvanzanten** [comment]
Would it be a temporary directus install, or should we come up with a more dedicated "development tool" a la storybook?

**Reply by rijkvanzanten** [reply]
Would be nice if you didn't have to go through the app to setup options etc, but just be able to render / test it inline

**Reply by maartenvn** [reply]
> Would be nice if you didn't have to go through the app to setup options etc, but just be able to render / test it inline

Yeah, this is what I mean. Just a temporary Directus install. Maybe a .directus folder and some command magic to compile & copy to that install. Everything under a single command (like `directus dev` or something)

**Reply by mdocter** [reply]
Great idea Maarten! Or start with some some basic extension scaffolding like `directus create interface my-interface` or `directus generate module my-module`.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
